tokens.xml

```diff
@@ -1,10 +1,10 @@
-<?xml version="1.0" encoding="UTF-8"?>
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
what's this change for?

Reply:
I think it was my lxml lib version when i runned the script to grab the new tokens

Reply:
For whatever reason, lxml defaults to single quotes in the header (probably because Python makes no distinction between single and double quotes) and I can't figure out a way to change it. It messes around the order of attributes in <cockatrice_carddatabase> too.

Reply:
Next time I will revert it back to double quotes and the correct order of the attributes as well